fetch.bs

```diff
@@ -3811,6 +3814,54 @@ the request.
  <var>request</var>'s <a for=request>origin</a> to  <var>request</var>'s <a for=request>client</a>'s
  <a for="environment settings object">origin</a>.
 
+ <li><p>Let <var>hasCustomHeaders</var> be false.
+
+ <li><p><a for=list>For each</a> <var>header</var> in <var>request</var>'s
+ <a for=request>header list</a>, if <var>header</var>'s <a for=header>name</a> is
+ not `<code>If-Modified-Since</code>`, `<code>If-None-Match</code>`, `<code>If-Unmodified-Since</code>`,
+ `<code>If-Match</code>`, `<code>User-Agent</code>`, or `<code>Cache-Control</code>`, set
+ <var>hasCustomHeaders</var> to true.
```

Comment:
This would always end up with _hasCustomHeaders_ being true.

Reply:
How come?
Anyway, maybe we should avoid `hasCustomHeaders` at all, and Let `fetch()` set `allow preloads` to false if there are user-defined headers.

Reply:
I had misread it, but it also seems the requirement is different now. Any author-defined headers triggers it now. But doesn't that also apply to `XMLHttpRequest`?